packages/flame_tiled/lib/src/renderable_layers/renderable_layer.dart

```diff
@@ -19,12 +21,16 @@ abstract class RenderableLayer<T extends Layer> {
   /// The parent [Group] layer (if it exists)
   final GroupLayer? parent;
 
+  /// The [FilterQuality] that should be used by all the layers.
+  final FilterQuality filterQuality;
+
   RenderableLayer({
     required this.layer,
     required this.parent,
     required this.map,
     required this.destTileSize,
-  });
+    FilterQuality? filterQuality,
+  }) : filterQuality = filterQuality ?? FilterQuality.low;
```

Comment:
```suggestion
  }) : filterQuality = filterQuality ?? FilterQuality.none;
```

Reply:
If I remember correctly low was the previous default

Reply:
I see. But low ever since a couple of flutter versions ago produced different results than none. I am fine with not changing to not make a breaking change, but thinking that most maps will be pixel art ones, changin to none will make the default better

Reply:
Let's change it, since it most likely wont make it worse for anybody :)